core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -106,32 +93,11 @@ public BasicInvocationStage visitPutKeyValueCommand(InvocationContext ctx, PutKe
 
    @Override
    public BasicInvocationStage visitPutMapCommand(InvocationContext ctx, PutMapCommand command) throws Throwable {
-      // don't bother with a remote get for the PutMapCommand!
-      return invokeNext(ctx, command);
-   }
-
-   @Override
-   public BasicInvocationStage visitGetKeyValueCommand(InvocationContext ctx, GetKeyValueCommand command) throws Throwable {
-      return visitGetCommand(ctx, command);
-   }
-
-   @Override
-   public BasicInvocationStage visitGetCacheEntryCommand(InvocationContext ctx, GetCacheEntryCommand command)
-         throws Throwable {
-      return visitGetCommand(ctx, command);
-   }
-
-   private InvocationStage visitGetCommand(InvocationContext ctx, AbstractDataCommand command)
-         throws Throwable {
-      Object key = command.getKey();
-      CacheEntry entry = ctx.lookupEntry(key);
-      // If the cache entry has the value lock flag set, skip the remote get.
-      if (ctx.isOriginLocal() && valueIsMissing(entry)) {
-         if (readNeedsRemoteValue(ctx, command)) {
-            remoteGet(ctx, key, false, command);
-         }
+      // remote commands are replayed in EWI
```

Comment:
I meant that EWI.EWV already makes sure that all entries are wrapped, therefore there's no need to check that again here.